is_all_unique_elements.js

```diff
@@ -0,0 +1,16 @@
+
+const isAllUniqueElements = (array) => {
+  let isAllUnique = true;
+
```

Comment:
What do you think about returning true or false below instead of assigning these values to a boolean? This will help dry up the code.

```suggestion
```